utils.py

```diff
@@ -223,3 +223,12 @@ def get_profile_pic(bot, user_id, size):
     photo_file.download(custom_path=filename_jpg)
     filename_png = convert_jpg_to_png(filename_jpg, delete_jpg=True)
     return filename_png
+
+
+# Possessive apostrophe rule in english grammar
+def apostrophe(name):
+    first_name = name.split()[0]
```

Comment:
changing the variable `name` to `player` and then issuing `first_name = player.name.split()[0]` should solve the issue.